conda-forge.yml

```diff
@@ -1,9 +1,10 @@
-travis:
-  secure:
-    BINSTAR_TOKEN: AVAEOvgJRTbM1j5fpK4p4DxAZlCxMjFbuWgi4p3Sz1XpV51yL7rFD0xYo6Bie+JAfRZeja4mJ6rwm+LaAYA6Xvl536jYVFLS4WSw0yIdFgo+cOHp3ZPIEMhLN7Aag4IeyRyPoIAFcDAMNGyP/kJCK07+dUBSDRuHYqC0LjToz1HlaPmWC1VHWqHAu2ENGyoxa37Mzaehut/kQrh1qHBOzV4YZYipw9W9oRauLmulL7HYlUJKtfchb58ifakIo4EdhDX8c5oI9EetV+skoOSso9Gg9wQRb3XWQxup4D2M/dMJ7gYRpE6Umc7LMKK24ilSFS0nXjIj9jOkEjmUP5jDbf6dih8wlwgwsMNf9BP38hZ/bTak32TUKMEMwc23DbOjzkzWbanwtJ8eafY1GVdwWzctygYT2NoGoDAZKUQ5PHVa+Keyy6LQiz5boIY243q8odrG2o20gDvyrBxRT5/LBbE98IB23wr3VoCiV53Mir3swRCfRhV8VxqYp3UeC8szYjF+EsLAF8Gh4DX9OWsLoihppjR3Vcrl4LcH1Am6bBH4m1xJs8rOINP0ILBJTlJ00qH//WA/9451ZBeYcvWA50kWJ3ApjkmBhyny8XaJKanB12FEtcswcnepAld2ZCsKhPjZtRgDmh5dujB/wvKV31F5lbemsA0lZDFXto8oZpo=
 appveyor:
-  secure:
-    BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
+  secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
+compiler_stack: comp7
+max_py_ver: '37'
+max_r_ver: '35'
+travis:
+  secure: {BINSTAR_TOKEN: AVAEOvgJRTbM1j5fpK4p4DxAZlCxMjFbuWgi4p3Sz1XpV51yL7rFD0xYo6Bie+JAfRZeja4mJ6rwm+LaAYA6Xvl536jYVFLS4WSw0yIdFgo+cOHp3ZPIEMhLN7Aag4IeyRyPoIAFcDAMNGyP/kJCK07+dUBSDRuHYqC0LjToz1HlaPmWC1VHWqHAu2ENGyoxa37Mzaehut/kQrh1qHBOzV4YZYipw9W9oRauLmulL7HYlUJKtfchb58ifakIo4EdhDX8c5oI9EetV+skoOSso9Gg9wQRb3XWQxup4D2M/dMJ7gYRpE6Umc7LMKK24ilSFS0nXjIj9jOkEjmUP5jDbf6dih8wlwgwsMNf9BP38hZ/bTak32TUKMEMwc23DbOjzkzWbanwtJ8eafY1GVdwWzctygYT2NoGoDAZKUQ5PHVa+Keyy6LQiz5boIY243q8odrG2o20gDvyrBxRT5/LBbE98IB23wr3VoCiV53Mir3swRCfRhV8VxqYp3UeC8szYjF+EsLAF8Gh4DX9OWsLoihppjR3Vcrl4LcH1Am6bBH4m1xJs8rOINP0ILBJTlJ00qH//WA/9451ZBeYcvWA50kWJ3ApjkmBhyny8XaJKanB12FEtcswcnepAld2ZCsKhPjZtRgDmh5dujB/wvKV31F5lbemsA0lZDFXto8oZpo=}
 channels:
   targets:
-    - [conda-forge, dev]
+    - ["conda-forge", "dev"]
```

Comment:
It seems like the upload will be to the dev label on conda-forge. See http://conda-forge.org/docs/maintainer/conda_forge_yml.html#channels It should be fine.